examples/build.gradle

```diff
@@ -1,16 +1,193 @@
 description = 'OpenCensus Examples'
 
+buildscript {
+    repositories {
+        mavenCentral()
+        mavenLocal()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath 'ru.vyarus:gradle-animalsniffer-plugin:1.4.2'
```

Comment:
I'm not sure we need as much functionality in the examples directory as we have for the main project.  I think we should try to keep the build file as simple as possible to make it a good example for users who just want to depend on OpenCensus.

Checking the license is important, though.  I wonder if it is possible to check the example classes from the main project instead.

Reply:
@bogdandrutu @sebright what plugins do you suggest we keep in this standalone project? grpc only uses java/protobuf/idea/application (no signing, jacoco, findbugs and checkstyle). If we want to keep the build file minimal, we should remove them, too.

Reply:
I think that `java` and `application` would be enough for now.  `idea` would be fine, too, especially if it is only one line.  We can add testing back if we need to add tests later.